docs/explanation/replication.rst

```diff
@@ -0,0 +1,68 @@
+Remote Replication
+==================
+
+Cloud storage services (like Ceph) are responsible for persisting information irrespective of faults and
+failures in parts of the cluster. These faults could be temporary network failures, disk faults, power failures
+or even failure of multiple nodes. Remote replication is a mechanism used to replicate data to a remote storage
+cluster, typically located at a different geographical site to prevent complete outage in the event of a large
+enough fault like natural disaster.
+
+This guide covers the essential replication concepts for new Ceph users.
```

Comment:
Suggest to word it like `remote replication concepts` otherwise there's a chance it gets confused with regular pool replication. In fact if not already done I think having a note somewhere that points out that local and remote repl are different things would be worthwhile.

Also suggest to not refer to "new Ceph users", think this is valuable for everyone